code/game/atoms.dm

```diff
@@ -48,6 +48,8 @@ var/global/list/ghdel_profiling = list()
 	//var/harm_label_icon_state //Makes sense to have this, but I can't sprite. May be added later.
 	var/list/last_beamchecks // timings for beam checks.
 
+	var/vending_cat = null// subcategory for vending machines.

```

Comment:
When we implement custom vending machines it'll be easy to code an override of that var, so players can set up their own categories and prices. Think of this as a "default" var, for items that you want automatically added to the machine, like for instance, seeds.